src/lib/provable/test/bitwise.unit-test.ts

```diff
@@ -160,82 +170,6 @@ await equivalentAsync({ from: [maybeField], to: field }, { runs })(
     return proof.publicOutput;
   }
 );
-await equivalentAsync({ from: [maybeField], to: field }, { runs })(
```

Comment:
why are we getting rid of all these unit tests?

Reply:
https://github.com/o1-labs/o1js/pull/1767#discussion_r1768704766